src/BookReaderComponent/BookReaderComponent.js

```diff
@@ -4,59 +4,65 @@
 
 import { LitElement, html, css } from 'lit-element';
 
-import '../ItemNavigator/ItemNavigator.js';
-import '../BookNavigator/BookNavigator.js';
-
+import '@internetarchive/ia-item-navigator';
+import '../BookNavigator/book-navigator.js';
+import '@internetarchive/modal-manager';
+import { SharedResizeObserver } from '@internetarchive/shared-resize-observer';
 export class BookReader extends LitElement {
   static get properties() {
     return {
-      base64Json: { type: String },
+      item: { type: String },
       baseHost: { type: String },
+      fullscreen: { type: Boolean, reflect: true, attribute: true },
+      sharedObserver: { type: Object }
     };
   }
 
   constructor() {
     super();
-    this.base64Json = '';
+    this.item = undefined;
+    this.bookreader = undefined;
     this.baseHost = 'https://archive.org';
+    this.fullscreen = false;
+    this.modal = undefined;
+    this.sharedObserver = new SharedResizeObserver();
   }
 
   firstUpdated() {
-    this.fetchData();
+    this.createModal();
   }
 
-  /**
-   * Fetch metadata response from public metadata API
-   * convert response to base64 data
-   * set base64 data to props
-   */
-  async fetchData() {
-    const ocaid = new URLSearchParams(location.search).get('ocaid');
-    const response = await fetch(`${this.baseHost}/metadata/${ocaid}`);
-    const bookMetadata = await response.json();
-    const jsonBtoa = btoa(JSON.stringify(bookMetadata));
-    this.setBaseJSON(jsonBtoa);
+  /** Creates modal DOM & attaches to `<body>` */
+  createModal() {
+    this.modal = document.createElement(
```

Comment:
We can cross this bridge when we get to it, but in offshoot, we'll want to pass in the modal manager into BR instead of it creating its own.. same with the SharedRO.

Reply:
yep. saving for that moment.